tests/pki-rpminspect.yaml

```diff
@@ -11,11 +11,11 @@ annocheck:
     jobs:
         - hardened: --skip-lto --skip-cf-protection
 
-# Use Java 21 on Fedora 40+, otherwise use Java 17
+# https://github.com/rpminspect/rpminspect-data-fedora/blob/main/fedora.yaml#L572-L596
 javabytecode:
-    - fc39: 61
-    - fc40: 65
-    - default: 65
+    - fc39: 55
+    - fc40: 55
+    - default: 43
```

Comment:
Since these are the values defined for fedora, are they needed or can be removed here to get them?

Reply:
Looks like they can be removed.